_static/files/audit-event-examples/R4_read.json

```diff
@@ -0,0 +1,122 @@
+{
+    "resourceType": "AuditEvent",
+    "type": {
+        "system": "http://hl7.org/fhir/audit-event-type",
+        "code": "rest",
+        "display": "RESTful Operation"
+    },
+    "subtype": [
+        {
+            "system": "http://hl7.org/fhir/http-verb",
+            "code": "GET"
+        },
+        {
+            "system": "http://hl7.org/fhir/restful-interaction",
+            "code": "read"
+        }
+    ],
+    "action": "R",
+    "recorded": "2022-10-07T14:21:15.456+00:00",
+    "outcome": "0",
+    "outcomeDesc": "200",
+    "agent": [
+        {
+            "altId": "unknown",
+            "who": {
```

Comment:
We should make sure that we adhere to https://profiles.ihe.net/ITI/BALP/content.html#35754-oauth-mapping-to-auditevent and map the iss and sub fields to agent.who correctly